git_greppers/README.md

```
@@ -0,0 +1,145 @@
+## Requirements
+
+- Bash shell environment
+- Git
+- GitHub CLI (`gh`) for GitHub repositories
+- GitLab CLI (`glab`) for GitLab repositories
+
+## Installation
+
+1. Clone or download this repository
+2. Make the scripts executable:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)